Tests/Functional/Traits/ProductInformationTrait.php

```diff
@@ -29,8 +30,9 @@ protected function createProductInformation(string $productId, string $locale):
             ]
         );
 
+        /** @var ProductInterface $product */
```

Comment:
whats the reason for this? also i dont think that type is right, cant this be null too? 🤔